src/vs/workbench/contrib/files/browser/views/explorerViewer.ts

```diff
@@ -417,6 +474,8 @@ export class FilesRenderer implements ICompressibleTreeRenderer<ExplorerItem, Fu
 		const realignNestedChildren = stat.nestedParent && themeIsUnhappyWithNesting;
 
 		templateData.label.setResource({ resource: stat.resource, name: label }, {
+			// We use null to indicate an explicit lack of tooltip vs undefined leaves it up to computation later in the rendering
+			title: isStringArray(label) ? label[0] : label,
```

Comment:
I don't see `null` as a type of `label` or `title`?

Reply:
Stale comment, removed.